src/pages/OfferNew/Perils/index.tsx

```diff
@@ -16,30 +15,25 @@ import {
 } from '../components'
 import { InsuranceValues } from './InsuranceValues'
 import { PerilCollection } from './PerilCollection'
-import { getLocalizedPerils } from './perilData'
 import { PerilModal } from './PerilModal'
 
 interface Props {
   insuranceType: InsuranceType
+  perils: ReadonlyArray<PerilV2>
 }
 
 const Wrapper = styled.div`
   padding: 8.5rem 0 5rem 0;
   background-color: ${colorsV2.offwhite};
   display: flex;
 `
+export const getIconUrl = (iconPath: string) =>
+  `https://graphql.dev.hedvigit.com${iconPath}` // TODO where should we get the base url from?
```

Comment:
I think we use `window.GIRAFFE_ENDPOINT` right now, but I do think that includes `/graphql`. Could we do something with that maybe?

Reply:
Updated now but it feels kinda hacky :D

Reply:
Very 😬 What about instead adding another environment variable?

Reply:
Hell yeah